cypress/support/fragments/users/users.js

```diff
@@ -38,4 +39,23 @@ export default {
     path: `bl-users/by-id/${userId}`,
     isDefaultSearchParamsRequired : false
   }),
+
+  createViaUi: (userData) => {
+    return cy.do([
+      Dropdown('Actions').find(Button()).click(),
+      Button({ id: 'clickable-newuser' }).click(),
+      TextField({ id: 'adduser_lastname' }).fillIn(userData.personal.lastName),
+      Select({ id: 'adduser_group' }).choose(userData.patronGroup),
+      TextField({ name: 'barcode' }).fillIn(userData.barcode),
+      TextField({ name: 'username' }).fillIn(userData.username),
+      TextField({ id: 'adduser_email' }).fillIn(userData.personal.email),
+      Button({ id: 'clickable-save' }).click()]).then(() => {
+      cy.intercept('/users').as('user');
+      return cy.wait('@user', { timeout: 30000 }).then((xhr) => xhr.response.body.id);
```

Comment:
please decrease the timeout to 3000-4000

Reply:
I tried to set the less waiter, but in this case there is not enough time and it gives an error, so I had to set the more value. The problem is that creating a user through UI. It takes a decent amount of time to process requests